.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1,4 +1,30 @@
 # zexi 0.4.0
+dynamodb_PartiQL:
+  title: Query a &DDB; table using PartiQL and &AWS; SDK 
+  title_abbrev: Query a &DDB; table using PartiQL
+  synopsis: query a &DDB; table.
+  category:
+  languages:
+    Kotlin:
+      versions:
+        - sdk_version: 1
+          github: kotlin/services/dynamodb
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.kotlin.partiql.main
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/dynamodb
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.java2.partiql.main
+  services:
+    dynamodb: {PartiQL}

```

Comment:
This list has to be actual service actions, so change to `ExecuteStatement`.

Reply:
fixed